src/pages/home/report/ReportActionItemMessageEdit.js

```diff
@@ -162,7 +162,7 @@ function ReportActionItemMessageEdit(props) {
      */
     const updateDraft = useCallback(
         (newDraftInput) => {
-            const {text: newDraft = '', emojis = []} = EmojiUtils.replaceEmojis(newDraftInput, props.preferredSkinTone, props.preferredLocale);
+            const {text: newDraft, emojis} = EmojiUtils.replaceAndExtractEmojis(newDraftInput, props.preferredSkinTone, props.preferredLocale);
 
```

Comment:
@parasharrajat I have added the above change. But are you trying to tell me to change `newDraft` to `convertedText` ?